system-addon/common/Actions.jsm

```diff
@@ -66,8 +66,8 @@ for (const type of [
   "TELEMETRY_PERFORMANCE_EVENT",
   "TELEMETRY_UNDESIRED_EVENT",
   "TELEMETRY_USER_EVENT",
-  "TOP_SITES_EDIT_CLOSE",
-  "TOP_SITES_EDIT_OPEN",
```

Comment:
^^ I had add these accidentally in a previous patch. They are telemetry event types not action types, so they don't belong here.